guides/doc-Release_Notes/topics/katello.adoc

```diff
@@ -41,7 +41,7 @@ Features performance improvements most notably around import/export for disconne
 
 === Pulpcore
 
-The pulpcore module stream must be disabled before updating packages. 
+The pulpcore module must be disabled before upgrading to Katello 4.11.
```

Comment:
Note to self: there is a step in the upgrade guide that https://github.com/theforeman/foreman-documentation/pull/2644/commits/43e1e32c905af403ca464b5ac90c2c6081567e76 adds again.